core/templates/dev/head/pages/exploration-player-page/learner-experience/conversation-skin.directive.ts

```diff
@@ -734,6 +737,14 @@ oppia.directive('conversationSkin', [
                     GLOBALS.collectionId, $scope.explorationId);
               }
 
+              if (ExplorationPlayerStateService.isInStoryViewerMode() &&
```

Comment:
Just one naming thing -- I think maybe you should call this "story chapter" mode instead of "story viewer" mode throughout. The "story viewer" is actually the page that lays out the explorations with the winding path through them (i.e. it's analogous to the collection viewer). But what you are trying to convey here is the idea that the exploration that's currently being played is a chapter of a story.

Reply:
Done.